liveblog/liveblog
# [LBSD-2810] - When ESI is switched on show more detailed notification in single blog embed settings

### Purpose

This PR improves the embed setting displayed for a blog when ESI is turned on for a particular theme.

### What has changed
- Included the ESI code in the embed code of the blog settings to allow easy copy-paste by editors.
- Added an external link to the Liveblog page for setting up ESI

### Steps to test
1. Create a new blog with a particular theme.
2. Head to the Theme Manager and go to the selected theme's settings. Toggle Advanced Settings and toggle the 'Optimise the Live Blog output for ESI' option.
3. Go back to the created blog and open it's settings.
4. Observe that the ESI code is displayed as well as an external link to the ESI setup guide.


### Screenshots

| Original  | New |
| ------------- | ------------- |
| ![image](https://github.com/liveblog/liveblog/assets/14116447/eb5a1dca-f08c-4f39-b3e2-0b05b71e1283) | ![image](https://github.com/liveblog/liveblog/assets/14116447/677ee377-ed1e-49ee-8356-f555e3b05efc) |


Resolves: [LBSD-2810](https://sofab.atlassian.net/browse/LBSD-2810)

[LBSD-2810]: https://sofab.atlassian.net/browse/LBSD-2810?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
